src/utils/getUrlNetworkParams.ts

```diff
@@ -0,0 +1,46 @@
+import { AvaNetwork } from '@/js/AvaNetwork'
+
+export const NETWORK_ALIAS = {
+    CAMINO: 'camino',
+    COLUMBUS: 'columbus',
+    KOPERNIKUS: 'kopernikus',
+    CUSTOM: 'custom',
+}
+
+export function getNetworkFromUrl() {
+    if (window.location.pathname.split('/')[1] === 'explorer') {
+        switch (window.location.pathname.split('/')[2]) {
+            case NETWORK_ALIAS.CAMINO:
+                return NETWORK_ALIAS.CAMINO
+            case NETWORK_ALIAS.COLUMBUS:
+                return NETWORK_ALIAS.COLUMBUS
+            case NETWORK_ALIAS.KOPERNIKUS:
+                return NETWORK_ALIAS.KOPERNIKUS
+            default:
+                return window.location.pathname.split('/')[2]
+        }
+    }
+    return ''
+}
+
+export function getCustomNetworks(): AvaNetwork[] {
```

Comment:
we already have the networks stored in the localStorage maybe if we use the same state from locaStorage will be good